src/rf/apps/workers/process.py

```diff
@@ -228,6 +228,7 @@ def thumbnail(self, record):
                                                enums.STATUS_THUMBNAIL)
 
         if make_thumbs_for_layer(layer_id):
+            log.info('Done generating thumbnail')
```

Comment:
Technically it is done generating thumbnail(s).

If we're going to have multiple workers we might not know which thumbnail job finished here. I think it is worth adding the layer id again. Agree/disagree?